addon/components/emoji-picker.js

```diff
@@ -90,14 +98,11 @@ export default Component.extend(ClickOutsideMixin, {
 
 
 
-  filteredEmojiCount: computed(
-    `emojiByCategoryIdFiltered.{${DEPENDENT_KEYS_FOR_EMOJI_CATEGORIES}}.length`,
-    function () {
-      return EMOJI_CATEGORIES_ARRAY.reduce((count, category) => {
-        return count + this.get(`emojiByCategoryIdFiltered.${category}.length`);
-      }, 0);
-    }
-  ),
+  filteredEmojiCount: computed('emojiByCategoryIdFiltered', function () {
```

Comment:
Removed the `DEPENDENT_KEYS_FOR_EMOJI_CATEGORIES` constant here.

The `emojiByCategoryIdFiltered` content is never changed except by creating a whole new `emojiByCategoryIdFiltered` object.